packages/components/src/components/as-histogram-widget/utils/draw.service.ts

```diff
@@ -155,7 +181,7 @@ export function renderYAxis(
 }
 
 function _delayFn(_d, i) {
-  return i * 50;
+  return i;
```

Comment:
Does this function do something?

Reply:
I get it. It's for starting animation depending on the number of the bucket, right?

Reply:
Exactly, it delays the animation by i millseconds. With i * 50 you get nice results, but if there's too many buckets it's kind of annoying.

With 'i' you don't notice unless there's a lot of buckets.